ibm/Dockerfile.j2

```diff
@@ -0,0 +1,33 @@
+FROM python:{{ BASE_IMAGE_VERSION }}-slim-buster
+
+LABEL maintainer="Maxime Sibellas <maxime.sibellas@ekino.com>"
+
+ARG TERRAFORM_VERSION
+ARG IBM_PROVIDER_VERSION
+ARG IBM_CLI_VERSION
+
+# ENV PATH=$PATH:/usr/local/bin
+
+WORKDIR /builds
+
+RUN pip install -U pip \
+    && pip install awscli boto3 pipenv
+
+RUN apt-get update -qq \
+    && apt-get -y -qq install wget \
+    && echo "Install IBMCloud CLI" && \
+    wget https://public.dhe.ibm.com/cloud/bluemix/cli/bluemix-cli/${IBM_CLI_VERSION}/binaries/IBM_Cloud_CLI_${IBM_CLI_VERSION}_linux_amd64.tgz && \
```

Comment:
Can you consistently put `&&` at the beginning or the end of the line?

Reply:
Fixed 👍